azure-pipelines.yml

```diff
@@ -11,12 +11,20 @@ stages:
     displayName: "Build and test"
     jobs:
       - job: build
+        strategy:
+          matrix:
+            Node10:
+              NODE_VERSION: '10.x'
```

Comment:
We still want to support Node10?  Isn't that way outdated now?  I still see it in `buildAndReleaseTask`, so I guess removing that should come separately either way though.